adapters/Vungle/VungleAdapter/GADMAdapterVungleInterstitial.m

```diff
@@ -32,6 +33,12 @@ @implementation GADMAdapterVungleInterstitial {
   BOOL _isAdLoaded;
 }
 
+// Redirect to the main adapter class for bidding
+// but still implement GADMAdNetworkAdapter for waterfall
++ (nonnull Class<GADMediationAdapter>)mainAdapterClass {
```

Comment:
Question: When will this be called? This is also in `GADMAdapterVungleRewardBasedVideoAd` but not `GADMAdapterVungleBanner`.

Reply:
This would be called by the Google SDK. From what I understand, the Google SDK looks for this legacy class when the publisher is trying to load an interstitial or banner and checks if this method is implemented. If it is implemented, the Google SDK would call this to get the adapter class and call the appropriate method for the ad format.

This is implemented in the `GADMAdapterVungleRewardBasedVideoAd` and here but not `GADMAdapterVungleBanner` because the Google SDK uses this class to load a banner as part of the older API.